frontend/src/components/modalContent/AddIncidentModal.js

```diff
@@ -64,11 +66,24 @@ const actionVerb = (newIncident) => {
   return <span>Edit&nbsp;</span>;
 };
 
-const StepForms = (props, next, prev, handleIncidentSubmit) => [
+const StepForms = (
+  props,
+  next,
+  prev,
+  handleIncidentSubmit,
+  uploadedFiles,
+  onFileLoad,
+  onRemoveFile
+) => [
   {
     title: "Initial Report",
     content: (
-      <AddIncidentReportingForm initialValues={props.initialValues} inspectors={props.inspectors} />
+      <AddIncidentReportingForm
+        initialValues={props.initialValues}
+        inspectors={props.inspectors}
+        onFileLoad={onFileLoad}
```

Comment:
Why is this prop passed here? We don't do file upload in this form?

Reply:
Removed 👍